altair/schema/_wrappers/encoding_channels.py

```diff
@@ -0,0 +1,82 @@
+# This file auto-generated by `generate_schema_interface.py`.
+# Do not modify this file directly.
+
+
+from .channel_wrappers import ChannelWithLegend
+from .channel_wrappers import PositionChannel
+from .channel_wrappers import Field
+from .channel_wrappers import Field
+from .channel_wrappers import OrderChannel
```

Comment:
Yep, missed that. We'll have to pass the imports to the templates as a single set rather than iterating through the imports of each object.